_posts/2013-04-06-developing-addons-and-blueprints.md

```diff
@@ -86,7 +86,10 @@ The addon project created follows these structure conventions:
 - `addon/` - part of the addon's namespace.
 - `blueprints/` - contains any blueprints that come with the addon, each in a separate directory
 - `public/` - static files which will be available in the application as `/your-addon/*`
-- `test-support/` - merged with the application's `tests/`
+- `test-support/` - merged with the application's `tests/` under the
+  application's namespace.
+- `addon-test-support/` - merged with the application's `tests/` uder the
```

Comment:
This might need some verbiage tweaking.

Can I suggest:

> available to the application during testing under addon's namespace at `${addon-name}/test-support/`